theme/theme_test.go

```diff
@@ -1,12 +1,21 @@
 package theme
 
 import (
+	"encoding/hex"
+	"fmt"
+	"image/color"
 	"testing"
 
 	"fyne.io/fyne"
 	"github.com/stretchr/testify/assert"
 )
 
+func helperColorToHexString(color color.Color) string {
```

Comment:
This is a duplicate of colorToHexString

Reply:
good catch, I forgot to remove that after a code restructure, fixed.